install/kubernetes/istio-rbac-alpha.yaml

```diff
@@ -24,6 +27,24 @@ rules:
   resources: ["namespaces", "nodes", "secrets"]
   verbs: ["get", "list", "watch"]
 ---
+# Mixer CRD needs to watch and list CRDs
+# It also uses discovery API to discover Kinds of config.istio.io
+# K8s adapter needs to list pods, services etc.
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+metadata:
+  name: istio-mixer
+rules:
+- apiGroups: ["config.istio.io"] # Istio CRD watcher
+  resources: ["*"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["configmaps", "endpoints", "pods", "services", "namespaces", "nodes", "secrets"]
```

Comment:
Does mixer need to read all these resources, including secrets? Can you try to reduce the list?

Reply:
yes, mixer will need secrets shortly as a way to get private credentials to adapters.